src/VisualStudio/Core/Def/Implementation/FindReferences/ContainingMemberColumnDefinition.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.ComponentModel.Composition;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Extensions;
+using Microsoft.VisualStudio.Composition;
+using Microsoft.VisualStudio.LanguageServices.FindUsages;
+using Microsoft.VisualStudio.Shell.TableControl;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.VisualStudio.LanguageServices.CustomColumn
+{
+    /// <summary>
+    /// Custom column to display the containing member for the All References window.
+    /// </summary>
+    [Export(typeof(ITableColumnDefinition))]
+    [Name(ColumnName)]
+    class ContainingMemberColumnDefinition : AbstractCustomColumnDefinition
+    {
+        public const string ColumnName = "ContainingMemberInfo";
+
+        [ImportingConstructor]
+        public ContainingMemberColumnDefinition()
+        {
+        }
+
+        public override bool IsFilterable => true;
+        public override string Name => ColumnName;
+        public override string DisplayName => ServicesVSResources.Containing_member;
+
+        public override string GetDisplayStringForColumnValues(ImmutableArray<string> values)
+        {
+            return values[0];
```

Comment:
Does this still accept multiple values and only display one? If not, which is my hunch, it should use `Single`

Reply:
I don't believe this code path is reachable for your current implementation and you should be able to just throw ExceptionUtilities.Unreachable here.

---
In reply to: [313616947](https://github.com/dotnet/roslyn/pull/37802#discussion_r313616947) [](ancestors = 313616947)